app/src/main/res/values/environment_config.xml

```diff
@@ -27,4 +27,6 @@
 
     <string name="notificationServiceBaseUrl">https://push-service.wallet.p2p.org/</string>
     <string name="notificationServiceTestBaseUrl">http://35.234.120.240:9090/</string>
+
+    <string name="googleClientId">553127941597-j88bgo2invnpiefq2j1fhq8duju6o2tq.apps.googleusercontent.com</string>
```

Comment:
looks like secret info for me, no?

Reply:
yes we should pass it somewhere in build config for example
what can you suggest?

Reply:
extract it to CI